### PR TITLE
[Access-tokens] Add thresholds

### PR DESCRIPTION
**Pull request by jprzychodzen**
/sig scalability
Ref. kubernetes/kubernetes#83375

### CODE REVIEW

File: sig-scalability/configs-and-limits/thresholds.md [hunk 1]
```diff
@@ -43,21 +43,23 @@ There are couple caveats to the thresholds we are presenting below:
...
+| #All service endpoints      | TBD                       | TBD                      |                              |
+| #Endpoints per service      | 250                       | n/a                      |                              |
+| #Secrets                    | TBD                       | TBD                      |                              |
+| #ConfigMaps                 | TBD                       | TBD                      |                              |
+| #Deployments                | 2000                      | TBD                      |                              |
+| #DaemonSets                 | TBD                       | TBD                      |                              |
+| #Jobs                       | TBD                       | TBD                      |                              |
+| #StatefulSets               | TBD                       | TBD                      |                              |
+| #AccessTokens               | 2000                      | 2000                     |                              |
+| #AccessTokens verifications | 5000                      | 5000                     | QPS, not a number of objects |
```
Comment (wojtek-t) [inline sig-scalability/configs-and-limits/thresholds.md]: Let's not add additional column just for this comment.

How about just making "5000 QPS" instead?

Comment (jprzychodzen) [reply sig-scalability/configs-and-limits/thresholds.md]: SG. Removed column and changed '5000' -> '5000 QPS'.

**Pull request merged**